src/nirizan/trust/attribution.py

```
@@ -26,9 +33,66 @@ class AttributionVerdict(BaseModel):
+    correction NiriZan's regression gate already relies on.
+    """
+
+    def __init__(
+        self,
+        alpha: float = 0.05,
+        confidence_level: float = 0.95,
+        n_bootstrap: int = 10000,
+        seed: int | None = None,
+    ) -> None:
+        if not (0.0 < alpha < 1.0):
+            raise ValueError("alpha must be strictly between 0 and 1.")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)